app/services/schools/advice/consumption_by_month_service.rb

```diff
@@ -38,6 +54,10 @@ def self.consumption_by_month(aggregate_meter)
                  end]
         end
       end
+
+      private_class_method def self.calculate_co2(consumption, fuel_type)
```

Comment:
This should be using code from `SecrCo2Equivalence.find_by!` code to find the appropriate factor for the calendar year for which we're calculating the monthly co2 value.